projects/redis.io/package.yml

```diff
@@ -0,0 +1,31 @@
+distributable:
+  url: https://download.redis.io/releases/redis-{{ version }}.tar.gz
+  strip-components: 1
+
+versions:
+  github: redis/redis
+
+
+provides:
+  - bin/redis-server
+  - bin/redis-cli
+  - bin/redis-benchmark
+
+dependencies:
+  openssl.org: ^1
+
+build:
+  dependencies:
+    gnu.org/make: '>=4.3'
+    freedesktop.org/pkg-config: '>=0.29'
+    tea.xyz/gx/cc: c99
+  script:
+    PREFIX={{prefix}}
+    BUILD_TLS=yes
+    make install
+    # TODO: configure default redis.conf if necessary
+test:
+  script: |
+    redis-server --test-memory 2
+    redis-cli --version
+    redis-benchmark --version
```

Comment:
Good number of tests. Two notes:

- if `script:` is the only subkey, you can pass the script directly on the `test:` key
- is it worth spinning up the server, running a couple of set/gets then spinning it down, to be sure it's working?

Reply:
added.